test/test.go

```diff
@@ -60,13 +62,20 @@ func TempFile(t *testing.T, dir string, pattern string, contents *string) *os.Fi
 	return file
 }
 
-func RecreateSymlink(t *testing.T, path string) error {
+// Lutimes is a convenience wrapper for using unix.Lutimes
+// TODO: this will need adapted if we support Windows.
+func Lutimes(t *testing.T, path string, atime time.Time, mtime time.Time) error {
 	t.Helper()
 
-	src, err := os.Readlink(path)
+	var utimes [2]unix.Timeval
+	utimes[0] = unix.NsecToTimeval(atime.UnixNano())
+	utimes[1] = unix.NsecToTimeval(mtime.UnixNano())
 
-	require.NoError(t, err, "failed to read symlink")
-	require.NoError(t, os.Remove(path), "failed to remove symlink")
+	// Change the timestamps of the symlink itself
```

Comment:
This comment mentions symlink, but I don't see any symlinks in this function?

Reply:
This is stale and needs removed.